appveyor.yml

```diff
@@ -0,0 +1,46 @@
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Set build version format here instead of in the admin panel
+version: "{build}"
+
+# Which branches to build
+branches:
+
+  # Blacklist
+  except:
+    - master
```

Comment:
We actually want to build master, I've disabled builds for branches that don't have an appveyor.yml so we can safely remove this I think.